guides/plugins/plugins/framework/message-queue/handle-asynchronous-messages.md

```diff
@@ -0,0 +1,61 @@
+---
+nav:
+  title: Handle asynchronous messages
+  position: 40
+
+---
+
+# Handle asynchronous messages
+
+## Overview
+
+::: warning
+This guide contains the `async_low_priority` queue which is only available in version 6.5.7.0 and above. You must not configure this queue in older versions as the messenger:consume command will fail.
```

Comment:
```suggestion
This guide contains the `async_low_priority` queue which is only available in version 6.5.7.0 and above. You must not configure this queue in older versions as the `messenger:consume` command will fail.
```

Reply:
I removed this file and refered to the `async_low_priority` in `add-messages-to-queue`.